From f3e54c0b85d6189d485503bc1368ce7bdb585130 Mon Sep 17 00:00:00 2001
From: Jordan Petridis <jpetridis@gnome.org>
Date: Tue, 12 May 2026 15:19:22 +0300
Subject: [PATCH] Update shell and mutter

---
 elements/core/gnome-shell.bst | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/elements/core/gnome-shell.bst b/elements/core/gnome-shell.bst
index 19b1995a..08dbfcbb 100644
--- a/elements/core/gnome-shell.bst
+++ b/elements/core/gnome-shell.bst
@@ -2,8 +2,8 @@ kind: meson
 
 sources:
 - kind: tar
-  url: gnome_downloads:gnome-shell/50/gnome-shell-50.0.tar.xz
-  ref: 85952fbd4929cfe613b876b2237e0d13464b0c2788b8a9867fb51c260255b0e3
+  url: gnome_downloads:gnome-shell/50/gnome-shell-50.1.tar.xz
+  ref: 1b47760172c14f3f4edd1c9aff365f4de45583517bf0f80df4d3acbd4e4cb294
 - kind: patch
   path: patches/gnome-shell/launch-installer.patch
 
-- 
2.54.0

